conf/app.routes

```diff
@@ -319,11 +319,11 @@ POST        /:taxYear/employment/expenses/start-employment-expenses
 GET         /:taxYear/student-loans/repayments                                       controllers.studentLoans.StudentLoansQuestionController.show(taxYear: Int, employmentId: String)
 POST        /:taxYear/student-loans/repayments                                       controllers.studentLoans.StudentLoansQuestionController.submit(taxYear: Int, employmentId: String)
 
-GET         /:taxYear/student-loans/postgraduate-repayment-amount                    controllers.studentLoans.PglAmountController.show(taxYear: Int, employmentId: String)
-POST        /:taxYear/student-loans/postgraduate-repayment-amount                    controllers.studentLoans.PglAmountController.submit(taxYear: Int, employmentId: String)
+GET         /:taxYear/student-loans/postgraduate-loan-repayment-amount               controllers.studentLoans.PglAmountController.show(taxYear: Int, employmentId: String)
+POST        /:taxYear/student-loans/postgraduate-loan-repayment-amount               controllers.studentLoans.PglAmountController.submit(taxYear: Int, employmentId: String)
 
-GET         /:taxYear/student-loans/undergraduate-repayment-amount                   controllers.studentLoans.UglAmountController.show(taxYear: Int, employmentId: String)
-POST        /:taxYear/student-loans/undergraduate-repayment-amount                   controllers.studentLoans.UglAmountController.submit(taxYear: Int, employmentId: String)
```

Comment:
Those changes will possibly fail the performance tests. Please ensure those are updated before merging

Reply:
Performance tests and journey tests are updated